OpenMarket/OpenMarket/Model/Network/OpenMarketApi.swift

```diff
@@ -6,26 +6,28 @@
 //
 
 import Foundation
-import UIKit
 
 enum OpenMarketApi {
     private static let hostUrl = "https://market-training.yagom-academy.kr/"
     
     case pageInformation(pageNo: Int, itemsPerPage: Int)
     case productDetail(productNumber: Int)
+    case productRegister
     
     private var urlString: String {
         switch self {
         case .pageInformation(let pageNo, let itemsPerPage):
             return Self.hostUrl + "api/products?page_no=\(pageNo)&items_per_page=\(itemsPerPage)"
         case .productDetail(let productNumber):
             return Self.hostUrl + "api/products/\(productNumber)"
+        case .productRegister:
+            return Self.hostUrl + "api/products"
         }
     }
     
     var url: URL? {
         switch self {
-        case .pageInformation, .productDetail:
+        case .pageInformation, .productDetail, .productRegister:
             let urlComponents = URLComponents(string: self.urlString)
             return urlComponents?.url ?? nil
         }
```

Comment:
앞으로 케이스가 추가되더라도 항상 같은 동작을 할 것 같은데 조금 더 간단하게 바꿔줄 수 있을 것 같아요.
`?? nil`도 불필요한 코드 같습니다.
```suggestion
    var url: URL? {
        let urlComponents = URLComponents(string: urlString)
        return urlComponents?.url
    }
```

Reply:
생각 해보니 그러네요. 바꿔주도록 하겠습니다!